homeassistant/components/gryfsmart/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""The Gryf Smart integration."""
+
+from __future__ import annotations
+
+import logging
+
+from pygryfsmart.api import GryfApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.discovery import async_load_platform
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_API, CONF_COMMUNICATION, CONF_DEVICE_DATA, CONF_PORT, DOMAIN
+from .schema import CONFIG_SCHEMA as SCHEMA
+
+CONFIG_SCHEMA = SCHEMA
+
+_PLATFORMS: list[Platform] = [
+    Platform.LIGHT,
+    Platform.BINARY_SENSOR,
+    Platform.SENSOR,
+    Platform.CLIMATE,
+    # Platform.COVER,
+    Platform.SWITCH,
+    # Platform.LOCK
+]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(
```

Comment:
this whole code should be migrated to using config entries.

Reply:
This is still unresolved